### PR TITLE
Create bio.md

### PR DESCRIPTION
**Pull request by krisnova**

### CODE REVIEW

File: content/bio.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Kris Nóva is a computer scientist, alpinist, author, public speaker and transgender advocate best known for her work on Linux and Kubernetes. 
+She specalizes in Linux kernel security, distributed systems and infrastructure management, and open source software engineering. 
+In 2017 she co-authored Cloud Native Infrastructure.
+
+She is well known for her open source contributions including projects like Linux, Kubernetes, and The Go Programming Language. 
+She is a popular public speaker, best known for her Kubernetes clusterfuck talks. She co-founded the cluster API project, created Kubicorn, and created naml for managing Kubernetes resources with pure Go.
+
+Today she works as a Senior Principal engineer building enterprise bare metal Kubernetes systems. 
+She is the founder of The Prvilege Escalation Foundation which is a non-profit focusing on offering resources for transgender minds in STEM.
```
Comment (justinliew) [inline content/bio.md]: s/Prvilege/Privilege

Comment (krisnova) [reply content/bio.md]: she knew that what she made would be the finest in the nation

Comment (justinliew) [reply content/bio.md]: 👜  🏷️  💸 🔪 🏬

**Pull request merged**